lib/librato/rails/configuration.rb

```diff
@@ -8,9 +8,9 @@ module Rails
     # https://github.com/librato/librato-rack/blob/master/lib/librato/rack/configuration.rb
     #
     class Configuration < Rack::Configuration
-      CONFIG_SETTABLE = %w{user token flush_interval log_level prefix source source_pids proxy suites}
```

Comment:
Thoughts on still allowing `source` as a config opt, but converting the value to tags? e.g., 

```ruby
{ hostname: "metrics-web-stg-1" }
```

We could log a deprecated warning as well, this should make transitioning easier by not rendering existing configs invalid for using `source`.